### DISCUSSION TITLE
Lighthouse deck: cannot communicate with lighthouse bootloader

### DISCUSSION THREAD

**Discussion by David-Glom** [bitcraze/discussions#1326] [Q&A]
Hi,
On one of our lighthouse decks I have the following problem:

![cannot communicate with lighthouse bootloader](https://github.com/bitcraze/discussions/assets/129954836/47052f68-610a-48ca-a0ae-961df83f7c5d)

It seems that the bootloader has a problem or that is not matching with the drone firmware. The reason has to be the lighthouse deck, because another lighthouse deck on the same drone works perfectly.

Is there a chance to flash the bootloader from the lighthouse deck? Because using the cold recovery function from cfclient only helps for the drone firmware but not with firmware from decks 

Thanks in advance!

**Comment by David-Glom** [comment]
I also resoldered very cautious the connector of the socket strip, to check if the drone can't communicate because of an invisible broken solder point, but this didn't help :/

**Reply by knmcguire** [reply]
ah man... that is a different issue... sorry that you seemed to be so unlucky with the lighthouse decks. I'll make sure to raise this with our meetings.

In terms of the lighthouse bootloader, you could take a look here? https://github.com/bitcraze/lighthouse-bootloader.  I'm not sure if this will work though but since you've already soldered something?

**Reply by David-Glom** [reply]
I will give it a try! Thanks :)

**Reply by David-Glom** [reply]
We have also eight working lighthouse decks, so some are also working :)

**Reply by knmcguire** [reply]
That is good to know... but we are a bit worried why you got 2 lighthouse deck in not working condition... so let's see if we can solve that!
